docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+.. py:currentmodule:: datatable
+
+Comparison with Rdatatable
+==========================
+
+``Datatable`` strives to mimic `Rdatatable's <https://rdatatable.gitlab.io/data.table/index.html>`_ API as closely as possible; however, there are differences due to language constraints.
+
+This page shows how to perform similar basic operations in `Rdatatable <https://rdatatable.gitlab.io/data.table/index.html>`_  in ``datatable``.
+
+Subsetting Rows
+----------------
+Sample data::
+
+    from datatable import dt, f, g, by, update, ifelse, join, sort
+    import numpy as np
+
+    # data is from the example data in Rdatatable
+
+    df = dt.Frame({"x":np.repeat(["b","a","c"],3),
+                   "y": [1,3,6]*3,
+                   "v": range(1,10)}) # datatable
+
+    DT = data.table(x=rep(c("b","a","c"),each=3),
+                    y=c(1,3,6), v=1:9) # rdatatable
```

Comment:
I would split this into two blocks of code: one for R and one for Python, because now it looks like this is a Python code that won't really work when executed. Then we could also use the same frame name `DT` in the both languages, so that it will look more consistent in the table below.